src/hooks/index.js

```diff
@@ -397,32 +462,46 @@ exports.restrictToCurrentSchool = async (context) => {
 	const currentSchoolId = user.schoolId.toString();
 	const { params } = context;
 	// route
-	if (params.route && params.route.schoolId && params.route.schoolId !== currentSchoolId) {
-		throw new Forbidden('You do not have valid permissions to access this.');
+	if (
+		params.route &&
+		params.route.schoolId &&
+		params.route.schoolId !== currentSchoolId
+	) {
+		throw new Forbidden(
+			'You do not have valid permissions to access this.',
+		);
 	}
 	// id
 	if (['update', 'patch', 'remove'].includes(context.method) && context.id) {
 		const target = await context.service.get(context.id);
 		if (!equalIds(target.schoolId, user.schoolId)) {
-			throw new NotFound(`no record found for id '${context.id.toString()}'`);
+			throw new NotFound(
+				`no record found for id '${context.id.toString()}'`,
+			);
 		}
 	}
 	// query
-	const methodWithQuery = ['get', 'find'].includes(context.method)
-		|| (['update', 'patch', 'remove'].includes(context.method) && context.id == null);
+	const methodWithQuery =
+		['get', 'find'].includes(context.method) ||
+		(['update', 'patch', 'remove'].includes(context.method) &&
```

Comment:
Codacy found an issue: ['&&' should be placed at the beginning of the line.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5994445)